marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/AbstractSparkConnectorTest.java

```diff
@@ -52,10 +52,26 @@ protected File getTestProjectDirectory() {
     @Override
     protected HubConfigImpl runAsUser(String username, String password) {
         hubProperties = new Properties();
-        // Assumes localhost for now; this will be improved once this project is developed on the 5.4 branch
-        hubProperties.setProperty("mlHost", "localhost");
+        String mlHost="localhost";
+        String hubDhs="false";
+        String hubSsl="false";
+        boolean isDhs=false;
+        //Can override if we want to run tests on DHS
+        if(System.getProperty("mlHost")!=null){
+            mlHost=System.getProperty("mlHost");
+        }
+        if(System.getProperty("isDhs")!=null) {
+            isDhs = Boolean.parseBoolean(System.getProperty("isDhs"));
+        }
+        if(isDhs){
+            hubDhs="true";
+            hubSsl="true";
+        }
+        hubProperties.setProperty("mlHost",mlHost);
         hubProperties.setProperty("mlUsername", username);
         hubProperties.setProperty("mlPassword", password);
+        hubProperties.setProperty("hubDHS",hubDhs);
```

Comment:
This should be "hubDhs", right? I believe "hubDHS" will still work because it gets lowercased, but we should still use hubDhs.